### ISSUE TITLE
[client-common] Missing `rows_before_limit_at_least` property in `ResponseJSON<T>` type

### ISSUE THREAD

**Issue by 0237h** [ClickHouse/clickhouse-js#267]
### Describe the bug

It seems that the property `rows_before_limit_at_least` (described [here](https://clickhouse.com/docs/en/interfaces/formats#json) in the docs) is missing from the `ResponseJSON<T>` type for the `client-common` library.

This leads to a `property doesn't exists` error in Typescript although the property can be accessed at runtime.

### Current definition
[packages/client-common/src/clickhouse_types.ts](https://github.com/ClickHouse/clickhouse-js/blob/a357f281d890a5569b3e446bc458d8301313f658/packages/client-common/src/clickhouse_types.ts#L1)
```ts
export interface ResponseJSON<T = unknown> {
    data: Array<T>;
    query_id?: string;
    totals?: Record<string, number>;
    extremes?: Record<string, any>;
    meta?: Array<{
        name: string;
        type: string;
    }>;
    statistics?: {
        elapsed: number;
        rows_read: number;
        bytes_read: number;
    };
    rows?: number;
}
```

### Expected definition

```ts
export interface ResponseJSON<T = unknown> {
    data: Array<T>;
    query_id?: string;
    totals?: Record<string, number>;
    extremes?: Record<string, any>;
    meta?: Array<{
        name: string;
        type: string;
    }>;
    statistics?: {
        elapsed: number;
        rows_read: number;
        bytes_read: number;
    };
    rows?: number;
    rows_before_limit_at_least?: number;
}
```

### Code example

```ts
const response = await client.query({ query, query_params, format: "JSON" });
const data: ResponseJSON<T> = await response.json();

console.log({
    statistics: data.statistics,
    rows: data.rows,
    rows_before_limit_at_least: data.rows_before_limit_at_least // property doesn't exists
});
```

#### Environment
* Client version: 1.0.1
* Language version: 5.4.5

**Comment by 0237h** [comment]
> Thanks for the report. Do you want to open a PR?

Sure can do :+1:

**Comment by slvrtrn** [comment]
The fix will be released as 1.0.2 on Monday.
